admin/spec/service/completed-check-processing.service.spec.js

```diff
@@ -24,7 +25,7 @@ describe('completedCheckProcessingService', () => {
         './data-access/completed-check.data.service': completedCheckDataService,
         './marking.service': markingService
       })
-      sandbox.stub(console, 'log')
+      sandbox.stub(winston, 'log')
```

Comment:
is log an actual method on winston? or has someone forgotten to change it to info/debug/error ?

Reply:
winston _does_ have a method called log, but I have now changed this to stub out `info` as this is clearer and what was actually intended.  Good spot, thanks.